### DISCUSSION TITLE
What is the current cost of training?

### DISCUSSION THREAD

**Discussion by LinB203** [PKU-YuanGroup/Open-Sora-Plan#20] [Q&A]
We use 8-frame 128×128 videos with a total batch size of 128 across 8 A100 GPUs, with a per-GPU memory usage of 48GB/80GB. Please note that we have not used any optimizations, such as mixed precision, flash-attention, or gradient checkpointing.

**Comment by ttttt12345** [comment]
I have lots of money beach.Suck my dick
